ports/nrf/Makefile

```diff
@@ -159,7 +159,6 @@ SRC_C += \
 	boards/$(BOARD)/pins.c \
 	device/$(MCU_VARIANT)/startup_$(MCU_SUB_VARIANT).c \
 	bluetooth/ble_drv.c \
-	bluetooth/ble_uart.c \
```

Comment:
Probably yes, I was just keeping it for reference.

Reply:
Changed my mind. It has support for REPL over BLE, so let's just leave it for now, for reference. It may get replaced or revised later -- I'm not sure yet.